src/mmw/js/src/core/settings.js

```diff
@@ -4,6 +4,8 @@ var _ = require('lodash');
 
 var defaultSettings = {
     itsi_embed: false,
+    itsi_enabled: true,
+    analyze_enabled: true,
```

Comment:
On one hand I like that these are separate top level variables, because there can be multiple reasons for disabling ITSI or Analyze. On the other hand, I'd prefer it if all the BIGCZ variables were grouped together in an object. Curious what others think.